CSharp/Library/Microsoft.Bot.Builder/Resource/Resources.resx

```diff
@@ -356,4 +356,19 @@
   <data name="UserProfileDeleted" xml:space="preserve">
     <value>User profile deleted!</value>
   </data>
+  <data name="NumberExpression" xml:space="preserve">
+    <value>[+-]?(?:\d+\.?\d*|\d*\.?\d+)</value>
+  </data>
+  <data name="NumberOrdinals" xml:space="preserve">
+    <value>1=1st,first|2=2nd,second|3=3rd,third|4=4th,fourth|5=5th,fifth|6=6th,sixth|7=7th,seventh|8=8th,eigth|9=9th,ninth|10=10th,tenth</value>
+  </data>
+  <data name="NumberReverseOrdinals" xml:space="preserve">
+    <value>-1=last|-2=next to last,second to last,second from last|-3=third to last,third from last|-4=fourth to last,fourth from last|-5=fifth to last,fith from last</value>
+  </data>
+  <data name="BooleanChoices" xml:space="preserve">
+    <value>true=y,yes,yep,sure,ok,\u1f44d,\u1f44c|false=n,no,nope,\u1f44e,\\u270b,\\u1f590</value>
+  </data>
+  <data name="NumberTerms" xml:space="preserve">
+    <value>0=zero|1=one|2=two|3=three|4=four|5=five|6=six|7=seven|8=eight|9=nine|10=ten|11=eleven|12=twelve|13=thirteen|14=fourteen|15=fifteen|16=sixteen|17=seventeen|18=eighteen|19=nineteen|20=twenty</value>
+  </data>

```

Comment:
I'm not sure; there is an open thread about using Steve's recognizers in C# vs porting LUIS recognizer to TypeScript but I don't think a decision was made yet.